test/functional/rpc_blockchain.py

```diff
@@ -388,6 +388,17 @@ def _test_gettxoutsetinfo(self):
         # Unknown hash_type raises an error
         assert_raises_rpc_error(-8, "'foo hash' is not a valid hash_type", node.gettxoutsetinfo, "foo hash")
 
+        self.restart_node(
+            0,
+            extra_args=[
+                "-coinstatsindex=1",
+                "-reindex"
+            ],
+        )
+        # Block not found error
```

Comment:
In 341d5e1da1a40ed6dec449bc64132de5ae8788c2, I don't think this comment is necessary since the next line is clear and "self explanatory": 
```suggestion
```

Reply:
thank you fixed in f169562d85b332576b3a1cf32755e43023227ed9